### DISCUSSION TITLE
Rate too high, when does it end?

### DISCUSSION THREAD

**Discussion by A380Coding** [jomjol/AI-on-the-edge-device#1854] [Q&A]
In case a value is read incorrectly and leads to a too high change rate. When does this situation end?
Is it necessary to manually set the previous value? Or is there a time out to this situation?

**Comment by caco3** [comment]
you have to set a new prevalue by yourself.

**Reply by A380Coding** [reply]
I'm reading the values via REST API and might not even notice when this happens and not get any further values.
What would be the recommended way besides setting a new prevalue? As a one time activity this is totally fine. Seems not to be right to permanently watch and manually reset the prevalue.

**Reply by caco3** [reply]
which REST API do you use?
If you use the `/json` you would get informed.

I agree that it would be good to have a better handling...

**Reply by A380Coding** [reply]
Using the /json API. I get informed that there is an error and there is no update value. But that still requires manual checking.
Can I set the prevalue via REST API? I haven't found anything on that. That might be something helpfull to automatically set from outside.

**Reply by caco3** [reply]
You can do it this way: `http://<IP>/setPreValue?value=12345&numbers=main`

**Reply by A380Coding** [reply]
Thanks! I will try that.

**Comment by korneij** [comment]
Hey guys,

I'm totally new & maybe asking a stupid question. I've crawled through the documentation & community here but not finally getting an answer to my question. I've set up this project for my watermeter, which worked out pretty easy and well. Even more surprised, that it's a real struggle with the gasmeter. 

I did everything like w/ the water; but it's permanently showing the status "rate too high".. 
* I tried to adjust the ROI better 
* rotated the ESP32 cam as good as possible to avoid the LED light on the numbers
* set the previous value manually

I then sent via MQTT the values once and then continues with this error. When rebooting the device it sometimes send the new value - but not guaranteed.

Does anyone have any further ideas?

![image](https://user-images.githubusercontent.com/123952756/215472885-feb44746-4c17-4fd8-9cc2-e7f25727c12e.png)

**Reply by caco3** [reply]
`rate too high` means that the change per round is higher than `MaxRateValue`.
You need to adjust this in the config.
